apps/recnet/src/server/routers/article.ts

```diff
@@ -12,13 +13,24 @@ export const articleRouter = router({
     .input(getArticlesParamsSchema)
     .output(getArticlesResponseSchema)
     .query(async (opts) => {
-      const { link } = opts.input;
+      const { link, useDigitalLibraryFallback: useDigitalLibraryFallback } =
+        opts.input;
       const { recnetApi } = opts.ctx;
       const { data } = await recnetApi.get(`/articles`, {
         params: {
           link,
+          useDigitalLibrary: useDigitalLibraryFallback,
```

Comment:
```suggestion
          useDigitalLibraryFallback,
```
If the param name has been changed to `useDigitalLibraryFallback`, why do we still have `useDigitalLibrary` here?

Reply:
I've tested your code by running applications locally, this caused bug. Since the backend expect the param name be `useDigitalLibraryFallback` and you pass `useDigitalLibrary`, backend won't be able to receive the boolean value you want to pass.

I added the code here to test
![Screenshot 2025-02-13 at 9 16 08 PM](https://github.com/user-attachments/assets/544a9149-450e-47d1-826b-bc121236cfc4)
The log from `recnet-api`
![Screenshot 2025-02-13 at 9 16 47 PM](https://github.com/user-attachments/assets/faf19c2e-3ebd-41f9-8fb2-9a2ddbd6af9e)
Please test your code and make sure everything works correctly.